### DISCUSSION TITLE
bounced emails list

### DISCUSSION THREAD

**Discussion by rizwany** [osTicket/osTicket#6252] [Ideas]
Actually we are using osticket for custom support, whenever user interacted with us on our site, it generates a ticket.

sometime user registers with invalid email, that generates bounce email ticket in osticket, so i want to extract those bounce ticket, and get  invalid emails from them, so i can mark those user invalid in our app database and no more send them an email.

Can we export list of emails which were bounced using osticket?

so if this can be done via any plugin or if you can suggest a table in osticket database to pull this info will be helpful

**Comment by ntozier** [comment]
This does not appear to be a bug report so I would recommend that you ask this question on the support forums.
https://forum.osticket.com/

Additionally, you will want to include a better description of what you are asking for.
Are you asking for a listing of emails sent from osTicket that resulted in a bounce messages (generated by a mail server)?
Are you saying that you only allow registered people to open tickets and want to know who emailed which did not result in a ticket being made?
Are you asking for something else?
